ynr/apps/sopn_parsing/tests/test_parse_tables.py

```diff
@@ -84,6 +89,32 @@ def test_basic_parsing(self):
             ],
         )
 
+        def test_values_contain_header_like_strings(self):
```

Comment:
The alignment is off here, so I don't think the test runner is actually picking up the test and running it, instead it just seems it as a function within `test_basic_parsing` (which is why `black` also doesn't pick this up).

If you indent it back so that it is inline with the other tests in this class it should then be run.